fluent-package/convert-artifacts-layout.sh

```diff
@@ -18,14 +18,20 @@ case $1 in
 	for d in buster bullseye bionic focal jammy; do
 	    case $d in
 		buster|bullseye)
-		    # e.g. mapping debian/pool/buster/main/t/td-agent/ => 4/debian/buster/pool/contrib/t/td-agent
-		    mkdir -p $ARTIFACTS_DIR/4/debian/$d/pool/contrib/t/td-agent
-		    find $REPOSITORY_PATH/debian/pool/$d -name '*.deb' -not -name '*dbgsym*' -exec cp {} $ARTIFACTS_DIR/4/debian/$d/pool/contrib/t/td-agent \;
+		    # e.g. mapping debian/pool/buster/main/t/td-agent/ => 5/debian/buster/pool/contrib/t/td-agent
+		    #      mapping debian/pool/buster/main/f/fluent-package/ => 5/debian/buster/pool/contrib/f/fluent-package
+		    mkdir -p $ARTIFACTS_DIR/5/debian/$d/pool/contrib/t/td-agent
```

Comment:
Note: The versioning rule might be changed so that it might not be `5` for released version.
But it's not needed to apply in this PR since it's not changed yet.